guides/release-notes.md

```diff
@@ -7,6 +7,18 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### Updated API: Wix Billing Backend
+
+`wix-billing-backend`
+
+We made the following changes to the [Invoices API](https://www.wix.com/velo/reference/wix-billing-backend/invoices):
+
+ + Added a `locale` object to [`createInvoice()`](https://www.wix.com/velo/reference/wix-billing-backend/invoices/createinvoice). The `locale` object supports creating customer invoices to be displayed in custom languages.    
+
+ + Updated the list of supported IETF language subtags in the `locale.language` property returned by [`getInvoice()`](https://www.wix.com/velo/reference/wix-billing-backend/invoices/getinvoice).
```

Comment:
Clarify here that the previous four-letter codes were not what was actually returned by the function, and that we've updated the list to align with the actual returned format.